src/style.css

```diff
@@ -1,3 +1,140 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__list {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-top: 40px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-top: 4px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  margin: 0;
+  padding-top: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star-last-star {
+  padding-left: 17px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star {
+  background-image: url("./images/star-active.svg");
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  border-radius: 0.5px;
+  margin-right: 4px;
+
+}
+
+.stars__star-last-star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  border-radius: 0.5px;
+}
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/07679f87-d92f-4897-a7e0-26e22fe22fc2)

And no need to repeat all styles